Dockerfile

```diff
@@ -25,10 +27,9 @@ RUN groupadd -g 999 tester && \
 USER tester
 
 # Install terminus
-RUN git clone https://github.com/pantheon-systems/terminus.git ~/terminus
-RUN cd ~/terminus && git checkout 3.x && composer install
-# Terminus 3 binary current is named t3.
-RUN ln -s ~/terminus/bin/t3 /usr/local/bin/terminus
+RUN curl -L https://github.com/pantheon-systems/terminus/releases/download/3.0.3/terminus.phar -o /usr/local/bin/terminus && \
```

Comment:
Use `ARG` for `TERMINUS_VERSION`?

Reply:
This is a Dockerfile.  Without the `ARG`, you would have to edit this file for every new release.  Just sayin'...

Reply:
Nope. We're starting from a fixed version and then run self:update. So, every time the image is built, it will get the latest terminus